src/main/java/teammates/common/util/StringHelper.java

```diff
@@ -507,5 +507,27 @@ public static int countEmptyStrings(String... strings) {
      */
     public static String convertToEmptyStringIfNull(String str) {
         return (str == null) ? "" : str;
-    }  
+    }
+    
+    /**
+     * Removes the outermost enclosing square brackets surrounding a string.
+     * 
+     * @param str
+     * @return the string without the outermost enclosing square brackets, 
+     *              if the given string is enclosed by square brackets <br/> 
+     *         the string itself if the given string is not enclosed by square brackets <br/>
+     *         null if the given string is null
+     */
+    public static String removeEnclosingSquareBrackets(String str) {
+        if (str == null) {
+            return null;
+        }
+        
+        String trimmedString = str.trim();
+        if (trimmedString.startsWith("[") && trimmedString.endsWith("]")) {
+            return trimmedString.substring(1, trimmedString.length() - 1).trim();
+        }
+        
+        return str;
```

Comment:
This is better. I prefer if you deal with the abnormal cases first, before you deal with the typical cases. i.e.
1. null
2. no brackets
3. typical case